frontend/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "env": {
+      "browser": true,
+      "jest": true,
+      "es6": true
+    },
+    "plugins": ["import"],
+    "extends": "eslint:recommended",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "rules": {
+      "no-console": "warn",
+      "no-eval": "error",
+      "import/first": "error"
+    }
+  }
```

Comment:
Like LinuxSuRen  mentioned in another review, space at the end of the file.

Reply:
Yep thanks keep forgetting it :P

Reply:
I would recommend switching to .eslintrc.js which then can be handled by eslint itself